apps/prairielearn/python/check_data.py

```diff
@@ -0,0 +1,168 @@
+from typing import Any, Dict, List, Literal, TypedDict
```

Comment:
Now that we're on Python 3.10, no need for uppercase `Dict`, `List`, or `Set` imports (lowercase versions work).

Reply:
Done in 8e121e4 / 0ec9590.